projects/packages/forms/src/contact-form/class-admin.php

```diff
@@ -862,8 +862,13 @@ public function grunion_source_filter_results( $query ) {
 			return;
 		}
 
-		// Don't apply to the filter dropdown query
-		if ( $query->query_vars['fields'] === 'id=>parent' ) {
```

Comment:
In https://github.com/Automattic/jetpack/pull/41602 we made the post type `hierarchical`. I tried to understand why this change affected this check the `fields` was `id=>parent` too, but couldn't figure it out. @anton-vlasenko do you have any thoughts?